package.json

```diff
@@ -60,6 +60,7 @@
         "@crawlee/memory-storage": "^3.5.8",
         "@oclif/command": "^1.8.36",
         "@oclif/config": "^1.18.17",
+        "@oclif/core": "^2.15.0",
```

Comment:
actually, since you modified the dependencies, i'd expect some lockfile changes too, are you sure there were none?

Reply:
Good point. Added.